### PR TITLE
Reformatting changes and some minor improvements 

### PR DESCRIPTION
**Pull request by vaithak**
The changes in the PR have been pulled apart from #171 for easy reviewal and better readability.

### CODE REVIEW

File: include/volume/math_helpers.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2021 Vaibhav Thakkar
```
Comment (vissarion) [inline include/volume/math_helpers.h]: please add the copyrights / names from the files you copied the functions that you pasted in this files e.g. `get_mean_variance`

File: include/volume/volume_cooling_gaussians.hpp [hunk 2]
```diff
@@ -23,6 +23,7 @@
 #include "random_walks/gaussian_ball_walk.hpp"
 #include "random_walks/gaussian_cdhr_walk.hpp"
 #include "sampling/random_point_generators.hpp"
+#include "math_helpers.h"
```
Comment (vissarion) [inline include/volume/volume_cooling_gaussians.hpp]: better `volume/math_helpers.h`

File: include/volume/math_helpers.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
+// VolEsti (volume computation and sampling library)
```
Comment (vissarion) [inline include/volume/math_helpers.h]: for consistency please use `.hpp` instead of `.h`

**Review by vissarion — APPROVED** [review]
Thanks for this PR. I have a few comments.

File: include/volume/volume_cooling_gaussians.hpp (hunk 2 above)
Comment (vaithak) [reply include/volume/volume_cooling_gaussians.hpp]: changed at all locations 👍🏽

File: include/volume/math_helpers.h (hunk 3 above)
Comment (vaithak) [reply include/volume/math_helpers.h]: renamed

File: include/volume/math_helpers.h (hunk 1 above)
Comment (vaithak) [reply include/volume/math_helpers.h]: added 👍🏽

**Pull request merged**